group.py

```diff
@@ -1,5 +1,60 @@
-"""An example of how to represent a group of acquaintances in Python."""
+group = {
+    "Jill": {
+        "age": 26,
+        "job": "biologist",
+        "relations": {
+            "Zalika": "friend",
+            "John": "partner"
+        }
+    },
+    "Zalika": {
+        "age": 28,
+        "job": "artist",
+        "relations": {
+            "Jill": "friend"
+        }
+    },
+    "John": {
+        "age": 27,
+        "job": "writer",
+        "relations": {
+            "Jill": "partner"
+        }
+    },
+    "NewGuyNoFriends": {
+        "age": 150,
+        "job": "student",
+        "relations": {
+        }
+    },
+    "Nash": {
+        "age": 34,
+        "job": "chef",
+        "relations": {
+            "John": "cousin",
+            "Zalika": "landlord"
+        }
+    }
+}
+
+# the maximum age of people in the group
+max_age = max([group.get(personName).get("age") for personName in group.keys()])
```

Comment:
You could simplify the base of your list comprehension a little bit by using the values directly
```suggestion
max_age = max(person.get("age") for person in group.values())
```